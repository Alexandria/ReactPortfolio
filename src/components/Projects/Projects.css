.Projects {
  height: inherit;
  width: inherit;
  background-color: cornflowerblue;
}
