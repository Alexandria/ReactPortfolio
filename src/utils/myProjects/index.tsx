import { Project } from "../../components/ProjectCard/ProjectCard";
import typescript from "../../assests/icons/typescript.png";
import Meeplely from "../../assests/images/Meeplely.png";
import nodeJs from "../../assests/icons/nodeJs.png";
import react from "../../assests/icons/react.png";
import postgres from "../../assests/icons/postgres.png";
import python from "../../assests/icons/python.png";
import phpmyadmin from "../../assests/icons/phpmyadmin.png";
import insomnia from "../../assests/icons/insomnia.png";
import django from "../../assests/icons/django.jpg";

export const myProjects: Project[] = [
  {
    title: "Meeplely",
    img: Meeplely,
    description:
      "Full stack web application that will help you manage your board game collection.",
    toolIcons: [typescript, nodeJs, react, postgres, insomnia]
  },
  {
    title: "Justice for Barb Blog",
    img: Meeplely,
    description:
      "Full stack web application that will help you manage your board game collection.",
    toolIcons: [django, python, phpmyadmin]
  }
];

export const project: Project = {
  title: "Meeplely",
  img: Meeplely,
  description:
    "Full stack web application that will help you manage your board game collection.",
  toolIcons: [typescript, nodeJs, react, postgres]
};
