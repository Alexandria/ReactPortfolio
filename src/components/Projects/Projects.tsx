import React from "react";
import "./Projects.css";
import typescript from "../../assests/icons/typescript.png";
import Meeplely from "../../assests/images/Meeplely.jpg";
import nodeJs from "../../assests/icons/nodeJs.png";
import react from "../../assests/icons/react.png";
import postgres from "../../assests/icons/postgres.png";
import { Project } from "../ProjectCard/ProjectCard";

interface Projects {
  projects: Project[];
}

const Projects: React.FC<Projects> = () => {
  return (
    <div className="card" style={{ maxWidth: "20rem" }}>
      <img src={Meeplely} height={300} alt="Meeple" />
      <div className="card-header">Meeplely</div>
      <div className="card-body">
        <blockquote className="blockquote mb-0">
          <p>
            Full stack web application that will help you manage your board game
            collection.
          </p>
          <footer>
            <img src={typescript} height={40} width={40} alt="Type Script" />
            <img src={nodeJs} height={40} width={40} alt="Node JS" />
            <img src={react} height={40} width={40} alt="react" />
            <img src={postgres} height={40} width={40} alt="postgres" />
          </footer>
        </blockquote>
      </div>
    </div>
  );
};

export default Projects;
