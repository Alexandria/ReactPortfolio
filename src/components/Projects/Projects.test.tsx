import React from "react";
import Projects from "./Projects";
import { shallowRender } from "../../utils/shallowRender";
import { myProjects } from "../../utils/myProjects";

describe("Projects", () => {
  it("renders expected structure", () => {
    const received = shallowRender(<Projects projects={myProjects} />);
    expect(received).toMatchSnapshot();
  });
});
