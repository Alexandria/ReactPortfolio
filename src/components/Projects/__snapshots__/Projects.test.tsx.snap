// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Projects renders expected structure 1`] = `
<div
  className="card"
  style={
    Object {
      "maxWidth": "20rem",
    }
  }
>
  <img
    alt="Meeple"
    height={300}
    src="Meeplely.jpg"
  />
  <div
    className="card-header"
  >
    Meeplely
  </div>
  <div
    className="card-body"
  >
    <blockquote
      className="blockquote mb-0"
    >
      <p>
        Full stack web application that will help you manage your board game collection.
      </p>
      <footer>
        <img
          alt="Type Script"
          height={40}
          src="typescript.png"
          width={40}
        />
        <img
          alt="Node JS"
          height={40}
          src="nodeJs.png"
          width={40}
        />
        <img
          alt="react"
          height={40}
          src="react.png"
          width={40}
        />
        <img
          alt="postgres"
          height={40}
          src="postgres.png"
          width={40}
        />
      </footer>
    </blockquote>
  </div>
</div>
`;
